src/mongocxx/exception/error_code.hpp

```diff
@@ -22,6 +22,7 @@ namespace mongocxx {
 MONGOCXX_INLINE_NAMESPACE_BEGIN
 
 enum class error_code : std::int32_t {
+	k_database_uninitialized = 1,
```

Comment:
'uninitialized' means something pretty specific in C++ land, and error codes are hard to change once you make them. Can we bikeshed a better name?